.travis.yml

```
@@ -13,7 +13,7 @@ install:
   - python -m pip --version
   - python -m tox --version
 script:
-  - python -m pip install .
+  - python -m pip install . --use-feature=in-tree-build
```

```suggestion
  - python -m pip install -e .
```
That would probably do the trick. I'm just not sure it's at all necessary. With `deploy` gone from travis. There is no need to do anything there.

Ideally I would suggest to remove it completely. Same with `appveyor`.

---

Yeah, nice catch, sadly I don't have the right on Travis either. At least it does not fail when doing nothing...